client/SDL/CMakeLists.txt

```diff
@@ -98,10 +98,25 @@ set(LIBS
         freerdp-client
     )
 
+option(WITH_WEBVIEW "Build with QtWebEngine support for AAD login popup browser" OFF)
+if (WITH_WEBVIEW)
+	include(FindPkgConfig)
+	pkg_check_modules(QT REQUIRED Qt5WebEngineWidgets)
+
+  list(APPEND SRCS sdl_webview.cpp)
+  add_definitions(-DWITH_WEBVIEW)
+  include_directories(${QT_INCLUDE_DIRS})
+  list(APPEND LIBS ${QT_LIBRARIES})
+endif()
+
 add_executable(${PROJECT_NAME}
 	${SRCS}
     )
 
+if (QT_FOUND)
+  target_compile_options(${PROJECT_NAME} PRIVATE -fPIC)
```

Comment:
that is already handled with `CMAKE_POSITION_INDEPENDENT_CODE` (or the corresponding target option)

Reply:
Apparently, cmake will only enable the -fPIE flag which is not good enough for qt

Reply:
`-v`? 
* I can compile perfectly without this line (and start `sdl-freerdp`)
* this option will only work with `gcc` and `clang` but fail with ms compilers
* https://stackoverflow.com/questions/15978608/qt-compile-your-code-with-fpic-or-fpie

Reply:
@fifthdegree but I´ve found the issue, using `find_package` with the `Qt::WebEngineWidgets` target properly sets all required options. When using `pkg-config` you have to manually set all of these, so I´ve pushed a fix here.